functional-tests/test-bed/pom.xml

```diff
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <scope>test</scope>
+      <scope>compile</scope>
```

Comment:
yes you  are right, but it's overriding from [parent](https://github.com/arquillian/smart-testing/blob/master/pom.xml#L89) where it's defined as `test` scope.